token/js/client/token.js

```diff
@@ -458,6 +481,91 @@ export class Token {
     return newAccount.publicKey;
   }
 
+  /**
+   * Create and initialize the associated account.
+   *
+   * This account may then be used as a `transfer()` or `approve()` destination
+   *
+   * @param owner User account that will own the new account
+   * @return Public key of the new associated account
+   */
+  async createAssociatedTokenAccount(owner: PublicKey): Promise<PublicKey> {
+    const associatedAddress = await Token.getAssociatedTokenAddress(
+      this.associatedProgramId,
+      this.programId,
+      this.publicKey,
+      owner,
+    );
+
+    return this.createAssociatedTokenAccountInternal(owner, associatedAddress);
+  }
+
+  async createAssociatedTokenAccountInternal(
+    owner: PublicKey,
+    associatedAddress: PublicKey,
+  ): Promise<PublicKey> {
+    await sendAndConfirmTransaction(
+      'CreateAssociatedTokenAccount',
+      this.connection,
+      new Transaction().add(
+        Token.createAssociatedTokenAccountInstruction(
+          this.associatedProgramId,
+          this.programId,
+          this.publicKey,
+          associatedAddress,
+          owner,
+          this.payer.publicKey,
+        ),
+      ),
+      this.payer,
+    );
+
+    return associatedAddress;
+  }
+
+  /**
+   * Get the associated account or create one if not found.
+   *
+   * This account may then be used as a `transfer()` or `approve()` destination
+   *
+   * @param owner User account that will own the new account
+   * @return Public key of the new associated account
+   */
+  async getOrCreateAssociatedAccountInfo(
```

Comment:
Some must-to-have helper.. I realized doing right like this is too hard for most of partners, I think...

Reply:
Yes, this is super useful!